js/projects.js

```diff
@@ -0,0 +1,191 @@
+//  Array storing all the projects
+const projects = [
+  {
+    name: 'Tonic',
+    description:
+        ' A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './assets/tonic-nature.png',
+    technologies: ['html', 'css', 'javascript'],
+    tags: ['CANOPY', 'Bac End Dev', '2015'],
+    link_live: 'https://kensteph.github.io/Portfolio/',
+    link_source: 'https://github.com/kensteph/Portfolio',
+  },
+  {
+    name: 'Multi-Post Stories',
```

Comment:
- 👏🏻 👏🏻 Excellent work implements an array of objects to store JavaScript objects containing the information for each project. 😉  💯